README.md

```diff
@@ -49,6 +49,9 @@ To write a blog:
 
 - create an author entry in [_data/authors.yaml](https://github.com/quarkusio/quarkusio.github.io/blob/develop/_data/authors.yaml)
   - `emailhash` you can get by running `echo your@email.org | md5` using an email you have registered from the [Gravatar service](https://gravatar.com),
+  
+     > Sometimes running `echo your@email.org | md5` doesn't generate the same hash used on Gravatar. You can also get the email hash directly in your profile on the Gravatar site.
```

Comment:
@edeandrea is it better with `echo -n your@email.org | md5sum`? Because from what I can see, it will always be incorrect without the `-n`.

Reply:
@gsmet `echo -n your@email.org | md5` seems to work (I'm not sure what `md5sum` does - but it seems to append some extra characters to the output).

I'll update the PR with this.

Reply:
@edeandrea are you on macOS?

Reply:
Yes.

I just updated the PR.